### PR TITLE
Add a refresh button in the header of comment thread

### LINKED ISSUE #5161: Add an icon button in Comment Thread widget to manually refresh the current thread
**Motivation** I regularly ran into such a problem -- somebody leaves a new comment in a thread from Github web interface, but that comment won't be timely revealed in VSCode until a few minutes later. I know that this extension is polling new updates in the background every several minutes, but it would be nicer to provide a choice for eager people to trigger an update right away.

Thus I propose to add a "refresh" icon button in the comment thread widget (for example, next to the "collapse all comments" icon), on which clicked, latest contents of the current thread are pulled eagerly at the moment. This idea is similar to the one in #4387 .

I am also interested in file a PR for this feature. But before that, I was wondering to know the team's attitude towards this feature. Are there any caveats on this we should care about?

### PR DESCRIPTION
**Pull request by hsfzxjy**
fix #5161

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2248,6 +2248,11 @@
 					"command": "pr.collapseAllComments",
 					"group": "collapse@0",
 					"when": "commentController =~ /^github-(browse|review)/"
+				},
+				{
+					"command": "pr.refreshComments",
+					"group": "refresh@0",
```
Comment (alexr00) [inline package.json:2254]: The order here seems wrong. I would keep this before the collapse action. 

image

**Review by alexr00 — CHANGES_REQUESTED** [review]
Thanks for the PR! I have one comment.

File: package.json (hunk 1 above)
Comment (alexr00) [reply package.json]: The order is determined by the `group`, not but the position in the `package.json`.

**Review by alexr00 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (hsfzxjy) [reply package.json]: Sorry for the delay. I've fixed it now.

**Review by alexr00 — APPROVED** [review]
Looks good, thanks!

**Review by aeschli — APPROVED** [review]

**Pull request merged**